frontend/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM node:10.15.3-stretch
 
 # AWS cli tools
+RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
```

Comment:
If you pull from master, we don't need this line anymore. I upgraded from stretch to buster.